src/ebonite/build/runner/simple_docker.py

```diff
@@ -1,75 +1,17 @@
-import re
```

Comment:
i think it's time to remove "simple" from filename

Reply:
Actually this code hasn't become any more complex. On the contrary some part of it moved to other module.